README.md

```diff
@@ -35,16 +35,31 @@ Simply update the files under `components/(team-name)`, and open a PR with the c
 
 ## Bootstrapping a cluster
 
+### Required prerequisites
 The prerequisites are:
 - You must have `kubectl` and `kustomize` installed. 
 - You must have `kubectl` pointing to an existing OpenShift cluster, that you wish to deploy to.
 
+### Optional: CodeReady Containers Setup
+If you don't already have a test OpenShift cluster available, CodeReady Containers is a popular option. It runs a small OpenShift cluster in a single VM on your local workstation.
+1) Create or log in using your free Red Hat account, and [install CodeReady Containers (CRC)](https://console.redhat.com/openshift/create/local).
+2) Make sure you have the latest version of CRC: `crc version`
+3) Set up your workstation and command line tools: `crc setup`
+4) Configure the VM using the minimum supported values. You can further increase these values if your workstation can support it: `crc config set memory 16384` and `crc config set cpus 6`
+5) Create a new VM after you adjust the memory and cpu allocation: `crc delete` and confirm with a `y`.
+6) Start the OpenShift cluster: `crc start` This command will output the OpenShift web console URL as well as the developer and kubeadmin credentials when it's finished.
+7) Set up your command line: `eval $(crc oc-env)`
+8) Configure kubectl to use the CRC administrator account: `kubectl config use-context crc-admin`
+
+### Bootstrap App Studio
 Steps:
-1) Run `hack/bootstrap-cluster.sh` which will bootstrap Argo CD (using OpenShift GitOps) and setup the Argo CD `Application` CRs for each component.
-2) Retrieve the Argo CD Web UI URL using `kubectl get routes`.
```

Comment:
The `kubectl get routes` command doesn't output the URL for me, but it is printed out at the end of the `bootstrap-cluster.sh` script.

Reply:
That's true, perhaps a picture is better for folks who are already logged into the cluster in the console. 
 
Navigate to the OpenShift Console and login to ArgoCD by selecting the OpenShift Gitops menu in the applications pulldown. 
<img width="596" alt="image" src="https://user-images.githubusercontent.com/7844190/143244105-ac617267-705a-441c-9977-ba0bb2e59460.png">

Reply:
Added.